app/components/ConfirmTrade.tsx

```diff
@@ -65,27 +77,92 @@ export const ConfirmTrade = ({
     ? formatEtherWithFixedDecimals(tokenAmountOut)
     : '';
 
+  const waitingTxToast = (txHash: string) =>
+    toast({
+      children: (
+        <div className="flex items-center space-x-4">
+          <Spinner className="h-5 w-5 shrink-0 animate-spin" />
+          <div>
+            Wating for transaction success{' '}
+            <a
+              href={`https://gnosisscan.io/tx/${txHash}`}
+              target="_blank"
+              className="inline-block underline"
+              rel="noopener noreferrer"
```

Comment:
```suggestion
```
I think it's safe to remove this `rel`, I think we already had a discussion where we found out that this is not a vulnerability anymore.